contracts/ERC165.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.23;
```

Comment:
what would you think of moving this to `contracts/introspection/ERC165.sol`

Reply:
sure